src/features/processPicker.ts

```diff
@@ -45,6 +46,27 @@ export class RemoteAttachPicker {
 
     private static _channel: vscode.OutputChannel = null;
 
+    public static ValidateAndFixPipeProgram(program: string): Promise<string> {
+        return PlatformInformation.GetCurrent().then(platformInfo => {
+            // If its 64 bit windows and the program does not exist. Try to replace System32 with sysnative
+            if (platformInfo.isWindows && platformInfo.architecture === "x86_64" && !fs.existsSync(program) && program.toLowerCase().includes("system32")) {
+                let sysRoot: string = process.env.SystemRoot;
+
+                // Escape backslashes for regex
+                let oldPath = path.join(sysRoot, 'System32').replace(/\\/g, '\\\\');
+                let newPath = path.join(sysRoot, 'sysnative');
+
+                // Regex to replace and ignore casing
+                let regex = RegExp(oldPath, "ig");
+
+                return program.replace(regex, newPath);
+            }
+
+            // Return original program and let it fall through
+            return program;
```

Comment:
It is possible the program does not exist in the sysnative path as well. Should we try and return the original program path if file wasn't found in sysnative path as well?